firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/common/Request.kt

```diff
@@ -65,3 +67,25 @@ internal data class CountTokensRequest(
       )
   }
 }
+
+@Serializable
+internal data class GenerateImageRequest(
+  val instances: List<ImagenPrompt>,
+  val parameters: ImagenParameters,
+) : Request {
+  @Serializable internal data class ImagenPrompt(val prompt: String)
+
+  @OptIn(PublicPreviewAPI::class)
+  @Serializable
+  internal data class ImagenParameters(
+    val sampleCount: Int?,
+    val includeRaiReason: Boolean?,
```

Comment:
Would it be ever so slightly safer to make these non-optional so we never accidentally set them to `null`? I see they're explicitly set above
```
GenerateImageRequest.ImagenParameters(
  sampleCount = config?.numberOfImages ?: 1,
  includeRaiReason = true,
```
so I don't think you'd need to change the call site at all now. If we decide to change the behaviour in the future it'll force us to think harder about it if non-nullable.